docs/Guides/Create-Business-Users.md

```diff
@@ -93,14 +93,14 @@ If another user already exists within your platform with the same email address
 ```
 
 
-## Coming soon
+## Update business locations for an existing user
+You may add and remove a user's accessible business locations in the following way.
```

Comment:
I think the add or remove language is a little misleading especially with the clarifying happening in next paragraph. Instead?:
```suggestion
You may update a user's accessible business locations in the following way.
```